src/libmongoc/src/mongoc/mongoc-util.c

```diff
@@ -889,9 +889,9 @@ _mongoc_simple_rand_uint64_t (void)
 
    /* Ensure *all* bits are random, as RAND_MAX is only required to be at least
     * 32767 (2^15). */
-   return (((uint64_t) rand () & 0x7FFFu) << 0u) | (((uint64_t) rand () & 0x7FFFu) << 15u) |
-          (((uint64_t) rand () & 0x7FFFu) << 30u) | (((uint64_t) rand () & 0x7FFFu) << 45u) |
-          (((uint64_t) rand () & 0x0003u) << 60u);
+   return (((uint64_t) rand_r (&_mongoc_simple_rand_seed) & 0x7FFFu) << 0u) | (((uint64_t) rand_r (&_mongoc_simple_rand_seed) & 0x7FFFu) << 15u) |
```

Comment:
To fix the failing clang-format task, run:
```sh
uv run tools/format.py
```

Reply:
Thanks! Though it does not nothing to me - https://pastila.nl/?00220984/992b50f96221d5337adc8cf28cc89af2#I1mfyuBcPkXinZfWfwWjpA==